cocos/base/CCEventDispatcher.cpp

```diff
@@ -619,7 +619,7 @@ void EventDispatcher::removeEventListener(EventListener* listener)
                 
                 if (_inDispatch == 0)
                 {
-                    listeners->erase(iter);
+                    iter = listeners->erase(iter);
```

Comment:
It's just for meet the C++11 standards, many 2dx user won't that good at C++ skills, I think give a sample in 2dx engine may help them decrease bug in coding.